server/src/tasks/eth_funder.js

```diff
@@ -5,7 +5,9 @@ const { createNetwork } = require('@utils/web3')
 const { toWei, toChecksumAddress } = require('web3-utils')
 const EthFunding = mongoose.model('EthFunding')
 
-const ethFunder = withAccount(async (account, { accountAddress }) => {
+const updateEthFunding = async (id) => EthFunding.findByIdAndUpdate(id, { funded: true })
```

Comment:
you should rename this to something like `setFunded` cause it's a very specific update